src/cluster/sparse_matrix_stream.h

```diff
@@ -106,6 +108,12 @@ class SparseMatrixStream : public Consumer {
 				const uint32_t subject = *(uint32_t*) ptr;
 				ptr += sizeof(uint32_t);
 				const double value = *(double*) ptr;
+				if(!warned && (value > std::numeric_limits<T>::max() || value < std::numeric_limits<T>::min())){
+					printf("\n");
```

Comment:
Please write to stderr since stdout is reserved for output.

Reply:
Will do, thanks.